### DISCUSSION TITLE
UI problem

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#813] [Q&A]
Hi, I followed the instruction download the VM client. I press the scan buttom but no respone and why can't I connect the crazyflies and the buttom is black I cannot to use ? tks.

**Comment by ataffanel** [comment]
We will need a lot more information to be able to help.

 - What operating system are you using on your host machine?
 - Did you manage to pass the Crazyradio dongle into the virtual machine?
 - Does the Crazyradio LED light-up red when you press the scann button?
 - What do you mean by "the button is black"? Can you post a screenshoot of the UI?

**Reply by CalvinKoung** [reply]
This problem I solve myself. But thanks your reply.
